src/Images.jsx

```diff
@@ -78,10 +78,8 @@ class Images extends React.Component {
         });
         utils.varlinkCall(utils.PODMAN, "io.podman.RemoveImage", {name: image})
                 .then((reply) => {
-                    const idDel = reply.image ? reply.image : "";
-                    const oldImages = this.props.images;
-                    let newImages = oldImages.filter(elm => elm.Id !== idDel);
-                    this.props.updateImages(newImages);
+                    this.props.updateImagesAfterEvent();
+                    this.props.updateContainersAfterEvent();
```

Comment:
Maybe I am missing something, but why the updateContainersAfterEvent is needed here? You delete some image, it suceeds, why you need to update containers?

Reply:
if the image has been deleted is used by a running container, that container will be deleted at the same time, so need to update the containers list.

Reply:
For this case you have to do force deletion and you have different handlers for that.
There is not posibility that some container will be using the image to be deleted and the deletion of the image will succeed without force option, so that you will get to these handlers here.